website/app/styles/components/component-api.scss

```diff
@@ -0,0 +1,160 @@
+// COMPONENT API
+
+// Property grid areas
+// | 1 | 3 | 5 |
+// | 2 | 4 | 5 |
+// | 6 | 6 | 6 |
+// | 7 | 7 | 7 |
+// | 8 | 8 | 8 |
+// | 9 | 9 | 9 |
+
+@use "../breakpoints" as breakpoint;
+@use "../typography/mixins";
+
+
+.doc-component-api {
+  margin-bottom: 24px;
+}
+
+.doc-component-api__property {
+  position: relative;
+  display: grid;
+  grid-template-rows: repeat(6, auto);
+  grid-template-columns: repeat(3, 1fr);
+  margin: 0;
+  padding: 24px 16px;
+  border-bottom: 1px solid #ccc;
+
+  @include breakpoint.small () {
+    display: block;
+  }
+}
+
+.doc-component-api__term {
+  @include doc-font-family("gilmer");
+  margin: 0;
+  margin-bottom: 8px;
+  padding: 0;
+  color: var(--doc-color-gray-300);
+  font-weight: 700;
+  font-size: 0.75rem;
+  line-height: 1.5;
+
+  &--name {
```

Comment:
I generally don't like chaining modifiers (for 'searchability' reasons, mainly) but in this case it would save us from quite a bit of code repetition